src/pages/index.module.scss

```
@@ -49,6 +50,21 @@ $color-background-stages-seconde: #51BCB3;
   }
 }
 
+.worldSkillsBanner {
+  background-color: $color-background-world-skills;
+  margin-top: $gap-between-sections;
```

```suggestion
.hero + .hero {
  margin-top: $gap-between-sections;
}
```
Plus de robustesse : on a pas un gap parceque c'est la bannière des WorldSkills, on a un gap parce que c'est une bannière qui suit une autre bannière

---

Et en fait je me rends compte que c'est le même gap sur toute la page, il y a probablement un meilleur sélecteur qui nous permettrait de se passer de la variable. Un truc du genre _lobotomised owl_ (`* + *`)